pkg/kymactl/cmd/version.go

```diff
@@ -42,7 +42,7 @@ func (o *VersionOptions) Run() error {
 	if version == "" {
 		version = "N/A"
 	}
-	fmt.Printf("Kymactl version:      %s\n", version)
+	fmt.Printf("Kyma CLI version:      %s\n", version)
```

Comment:
Amount of spaces should be reduced by 1 to have formatting aligned with server side version

Reply:
done